### DISCUSSION TITLE
What's the correct way to unload LORA

### DISCUSSION THREAD

**Discussion by FartyPants** [oobabooga/text-generation-webui#2232] [Q&A]
Here is a question, after applying LORA, what is the correct way to unload it? AKA, select NO LORA.
If I delete it in the box and then press Apply Selected LORA - it doesn't like that - it ends up in error.

**Comment by Nixellion** [comment]
I want to +1 this. So far seems like restarting the entire webui is required. I don't get an error, but whenever I 'refresh' the list of loras it shows up that they are all applied. Also from model's speed and responses it's apparent that they are still present.
